package.json

```diff
@@ -53,7 +53,38 @@
           "description": "Run the linter on save (onSave) or on type (onType)."
         }
       }
-    }
+    },
+    "languages": [
+      {
+        "id": "json",
+        "filenames": [
+          ".pug-lintrc",
+          ".jade-lintrc"
+        ]
+      }
+    ],
+    "jsonValidation": [
+      {
```

Comment:
I guess that's wildcard matching but not necessarily globbing.  Or, as you said, it's a limited form of globbing that doesn't support brackets.  I tested this morning with brackets and they didn't work.